components/ui/AppCard.vue

```diff
@@ -201,3 +207,56 @@ export default class AppCard extends Vue {
   }
 }
 </style>
+
+<style lang="scss">
+@import '~carbon-components/scss/globals/scss/typography';
+
+.app-card {
+  .bx--tag--purple {
+    background-color: $tag-background-color;
+    color: $tag-text-color;
+    margin-right: $spacing-03;
+    margin-left: 0;
+    min-width: 0;
+  }
+
+  .bx--tag--purple:last-child {
+    margin-right: 0;
+  }
+
+  &__custom-pill {
+    @include type-style('caption-01');
+
+    background-color: $tag-background-color;
+    color: $tag-text-color;
+    display: inline-flex;
+    min-width: 0;
+    max-width: 100%;
+    min-height: 1.5rem;
```

Comment:
Please, use for margin and padding the carbon spacing variables like you did [here](https://github.com/Qiskit/qiskit.org/pull/2660/files#diff-39a1c5dd7d7f4b0c976425ad09d8aae1069e5758cbe7a1f616550c1dc1e4ef1eR242)